src/eko/runner.py

```diff
@@ -165,36 +166,91 @@ def compute_msbar_mass(theory_card):
         masses: list
             list of msbar masses squared
     """
-    masses = np.full(3, np.inf)
-    nf_active = 3
+    if theory_card["nfref"] is None:
+        raise ValueError(
```

Comment:
I don't think this is really needed: if you have `Q_b < Q_ref < Q_t` I think you can assume `nf_ref=5`

Reply:
Yep, let's assume that by default we do the only possible thing.

If we won't to stress more the "defaultless" pattern, we'll impose it from the input layer (whenever there will be...)